### DISCUSSION TITLE
Annotating Images

### DISCUSSION THREAD

**Discussion by andrewcrook** [argenos/zotero-mdnotes#188] [General]
As far as I can see Zotero 6 allows you to select images as annotations. 
Converting annotations to notes works.
It is just the rendering of the markdown and included image files.
These could be exported to a folder relative to the markdown file. 
Some tools such as Obsidian have plugins that can move files and update links for media data 
so the user doesn’t have to stick with this arrangement.
May be each export should create its own folder structure contacting all the components.
Is it easy to pull the images out of notes?
I presume they are in a folder?
Does Zotero store its notes as markdown or rtf?

**Comment by argenos** [comment]
It might be possible (see https://github.com/argenos/zotero-mdnotes/issues/62), but I don't know when/if I'll be able to implement this any time soon. If you are using Obsidian, you might consider using the Zotero integrator plugin instead of Mdnotes.
